tox.ini

```diff
@@ -14,13 +14,17 @@ ignore=
 
 [testenv]
 usedevelop=True
+install_command = pip install {opts} --process-dependency-links {packages}
 passenv =
     PYTEST_ADDOPTS
     TRAVIS_EVENT_TYPE
 commands=
     benchmark: {toxinidir}/scripts/benchmark/run.py
     core: py.test {posargs:tests/core/}
     trinity: py.test {posargs:tests/trinity/core/}
+    # The SMC tests may fail as they depend on eth-tester which does not necessarily work with
+    # py-EVMs master branch.
```

Comment:
If tests are known to be failing, in a way that should change in a future release, I think the best way to communicate that is with an xfail.

In this case, something like: `@pytest.mark.xfail(reason="depends on recent eth-tester, which doesn't work with current py-evm")` on the test that's failing.